plutus-core/untyped-plutus-core/Language/UntypedPlutusCore/Core/Type.hs

```diff
@@ -92,6 +108,23 @@ termAnn (Delay ann _)    = ann
 termAnn (Force ann _)    = ann
 termAnn (Error ann)      = ann
 
+bindFun
+    :: (ann -> fun -> Term name uni fun' ann)
```

Comment:
1. [no](https://tech.fpcomplete.com/blog/2017/07/to-void-or-to-void)
2. we need that function to [use it](https://github.com/input-output-hk/plutus/blob/b27ed7debb7fbeaf84874485700d90f0f5baabe7/plutus-core/untyped-plutus-core/Language/UntypedPlutusCore/HOAS.hs#L192) over non-`Void` `fun'`

Reply:
> we need that function to use it over non-`Void` `fun'`

I.e. it doesn't get rid of all the built-in functions, it's just that in the updated term they are in `fun'` rather than `fun`.